packages/api/src/schema/schema.yaml

```diff
@@ -3022,17 +3086,26 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
 
-  /user/{id}:
+  "/user/{id}":
     patch:
```

Comment:
This API doesn't exist. IDK why it was added here lol

Reply:
I believe it does exist: https://github.com/livepeer/studio/blob/master/packages/api/src/schema/schema.yaml#L3025

Reply:
Please refer to my other comment. The schema is not the source of truth, only the code is.

And clarifying my original comment: 
With "IDK why it was added here lol" I actually meant idk why it had been added to this schema in the first place (not a change from this PR). Maybe the API existed at some point and was removed, IDK, but on the controller the only patch API I can find is [`PATCH /user/:id/suspended`](https://github.com/livepeer/livepeer-com/blob/cb8d298f5c1c6220c46fab07fbe39158070c2408/packages/api/src/controllers/user.ts#L409) which is also an admin-only API anyway, so should not be documented here.